easybuild/easyconfigs/q/Qtconsole/Qtconsole-5.0.2-foss-2020b.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'Qtconsole'
+version = '5.0.2'
+
+homepage = 'https://jupyter.org/'
+description = """A rich Qt-based console for working with Jupyter kernels, supporting rich media output, session export, and more.
+The Qtconsole is a very lightweight application that largely feels like a terminal, 
+but provides a number of enhancements only possible in a GUI, 
+such as inline figures, proper multiline editing with syntax highlighting, graphical calltips, and more."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

We've moved IPython to GCCcore, so this should be moved as well.

---

fixed in dae3e66368